src/main/kotlin/com/project/goms/domain/outing/presentation/OutingController.kt

```diff
@@ -46,4 +44,9 @@ class OutingController(
             .let { outingConverter.toSearchOutingResponse(it) }
             .let { ResponseEntity.ok(it) }
 
+    @GetMapping("validation")
+    fun validateOuting() : ResponseEntity<Boolean> =
```

Comment:
여기도 컨벤션을 맞추면 좋을거 같습니다.
```suggestion
    fun validateOuting(): ResponseEntity<Boolean> =
```

Reply:
수정했습니다.